definitions/variable/emissions/emissions.yaml

```diff
@@ -19,48 +19,82 @@
     unit: Mt CO2/yr
 
 - Emissions|{Level-2 Species}|AFOLU:
-    description: Emissions of {Level-2 Species} from agriculture, forestry
-      and other land use (IPCC category 3)
+    description: Emissions of {Level-2 Species} from agriculture, forestry and other land use (IPCC category 3).  
+      In particular, this includes agricultural emissions (mostly non-CO2) from livestock, rice cultivation and soil 
+      management as well as emission from land use and land-use change (mostly CO2) such as deforestation, 
+      conversion of non-forest natural land, forestry activity, agroforestry, Re/Afforestation and managed peatlands
     unit: "{Level-2 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture:
-    description: Emissions of {Level-3 Species} from the agriculture sector
+    description: Emissions of {Level-3 Species} from the agricultural sector
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture|Livestock:
-    description: Emissions of {Level-3 Species} from livestock in the agriculture sector
+    description: Emissions of {Level-3 Species} from livestock in the agricultural sector
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture|Livestock|Enteric Fermentation:
     description: Emissions of {Level-3 Species} from enteric fermentation of livestock
       in the agriculture sector
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture|Livestock|Manure Management:
     description: Emissions of {Level-3 Species} from processes involving
       manure (waste) management in Livestock in the agriculture sector
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture|Rice:
     description: Emissions of {Level-3 Species} from rice cultivation
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Agriculture|Managed Soils:
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Land:
-    description: Emissions of {Level-3 Species} from forestry and other land use
+    description: Emissions of {Level-3 Species} from land use and land-use change including deforestation, 
+      conversion of non-forest natural land, forestry activity, agroforestry, Re/Afforestation and managed peatlands
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
     description: Emissions of {Level-3 Species} from managed peatlands (drained and rewetted)
     unit: "{Level-3 Species}"
+    tier: 2
 - Emissions|{Level-3 Species}|AFOLU|Land|Forest Burning:
```

Comment:
The "burning" variables don't fit well under `Emissions|CO2|AFOLU|Land`, which in my understanding reflects CO2 emissions from land-use change. 
I suggest to add a "Burning" sub-category like `Emissions|{Level-3 Species}|AFOLU|Burning|Forest` for more clarity. 
As far as I'm aware, CO2 emission from land-use change and burning emissions are often reported from different modules / post-processing steps within IAMs. 
@jkikstra @gidden What do you think?